drv/ArrayList.drv

```diff
@@ -587,6 +580,40 @@ public class ARRAY_LIST KEY_GENERIC extends ABSTRACT_LIST KEY_GENERIC implements
 		return modified;
 	}
 
+#ifdef JDK_PRIMITIVE_PREDICATE	
+	/** Remove from this collection all elements which satisfy the given predicate.
+	 *
+	 * @param filter a predicate which returns {@code true} for elements to be
+	 *        removed.
+	 * @see Collection#removeIf(java.util.function.Predicate)
+	 * @return {@code true} if any elements were removed.
+	 */
+	public boolean remIf(final JDK_PRIMITIVE_PREDICATE filter) {
```

Comment:
Maybe `removeIfPrimitive`?

Reply:
Actually, nevermind. This should be `removeIf`, the primitive predicates do not extend the generic predicates, so there is no clash

Reply:
IntList.rem() to remove a primitive. So it is consistent with lists.
And when i tried using removeIf I have to cast because otherwise the compiler will loses its mind.
In that case it is better to split the function names from each other.

Reply:
Up for @vigna to decide.